roles/capi/tasks/main.yml

```diff
@@ -9,12 +36,23 @@
     owner: root
     group: root
 
+# see https://github.com/rancher-sandbox/cluster-api-provider-rke2#setting-up-clusterctl
+- name: Ensure  ~/.cluster-api directory exists
+  file:
+    path:  ~/.cluster-api
+    state: directory
+
+- name: Setup clusterctl.yaml
+  template:
+    src: clusterctl.yaml.j2
+    dest: ~/.cluster-api/clusterctl.yaml
+
```

Comment:
I'm using the --config clusterctl.yaml on the line 55 to remove this part from the script

Reply:
Ack either approach works - the main advantage of this approach is it removes the risk of someone forgetting to add `--config` (particularly if re-running `clusterctl` to upgrade in future) it could pull in the wrong/upstream images

That said we need to decide if a helm chart may be a better option for the day-2 upgrades, hence https://github.com/suse-edge/charts/pull/103 but it's currently still WIP

Reply:
ok super! thanks